pep-9999.rst

```diff
@@ -0,0 +1,210 @@
+PEP: 9999
+Title: Remove wstr from Unicode
+Author: Inada Naoki <songofacandy@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 24-Jun-2020
+Python-Version: 3.10
+
+
+Abstract
+========
+
+PEP 393 deprecated some unicode APIs, and introduced ``wchar_t *wstr``,
+and ``Py_ssize_t wstr_length`` in unicode implementation for backward
+compatibility of these deprecated APIs. [1]_
+
+This PEP is planning removal of ``wstr``, and ``wstr_length`` with
+deprecated APIs using these members.
```

Comment:
Please mention the Python version when last deprecated bits will be removed (Python 3.14 currently).